OpenEdXMobile/res/values/attrs.xml

```diff
@@ -23,6 +23,10 @@
         <attr name="contourSize" format="dimension"/>
     </declare-styleable>
 
+    <declare-styleable name="EdxDiscussionBody">
+        <attr name="bodyStyle" format="reference" />
```

Comment:
How about changing this attribute's name to something like the following?
1. `bodyTextStyle`
2. `textInBodyStyle`
3. `bodyTextAppearance`

cuz in essence, this style will only be applied to the text that'll be appearing inside the body of a discussion item.

Reply:
Updated to `bodyTextAppearance`.